force-app/main/default/classes/ClassesObjectsMethods.cls

```diff
@@ -378,6 +426,14 @@ public with sharing class ClassesObjectsMethods {
      * @return The Person object of the older person.
      */
     public static Person getOlderPerson(Person person1, Person person2) {
-        return null; // Replace null with the variable you used to store the result
+        if (person1.getAge() > person2.getAge()) {
+            return person1;
+        }
+        else if (person2.getAge() > person1.getAge()) {
+            return person2;
+        }
+        else {
+            return person1;
```

Comment:
I don't think you need that else if  condition. If person one is not greater then we have to return person 2.